integration-test/common/src/test/java/org/apache/uniffle/test/QuorumTest.java

```diff
@@ -427,18 +431,25 @@ public void case3() throws Exception {
     } catch (Exception e) {
       assertTrue(e.getMessage().startsWith("Get shuffle result is failed"));
     }
+    shuffleServers.set(1, createServer(1, tmpDir));
+    shuffleServers.get(1).start();
```

Comment:
It maybe the root cause of #483. @jerqi @zuston

Reply:
I think we should not use `static` here.
https://github.com/apache/incubator-uniffle/blob/d60d675d38c833b99b012a1f4c726a012ce93463/integration-test/common/src/test/java/org/apache/uniffle/test/IntegrationTestBase.java#L60-L61